Microsoft.DotNet.Interactive/Commands/InterruptKernel.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Microsoft.DotNet.Interactive.Commands
 {
-    public class InterruptExecution : KernelCommandBase
+    public class InterruptKernel : KernelCommandBase
```

Comment:
The thing being interrupted isn't the kernel. It's an operation or an invocation or something.